ci/infra/testrunner/testrunner

```diff
@@ -18,7 +19,12 @@ setup_python_env() {
   source ${VENVDIR}/bin/activate
   # Not calling to `pip3` directly because its shebang can exceed 128 characters
   # and this operation will fail. Check: https://github.com/pypa/pip/issues/1773
-  python -m pip install -r "${SDIR}/requirements.txt" &>/dev/null
+  if [[ $PIP_VERBOSE == "true" ]]; then
+    python -m pip install -r "${SDIR}/requirements.txt"
+  else
+    python -m pip install -r "${SDIR}/requirements.txt" &>/dev/null
```

Comment:
I'm not personally a huge fan of this bash-specific redirection, but if we can always assume this will be bash (as opposed to `/bin/sh`), it's fine. :)